nuxeo-vision-core/src/test/java/org/nuxeo/vision/core/test/TestPictureEventChain.java

```diff
@@ -76,6 +83,25 @@
 
     @Inject
     protected TagService tagService;
+    
+    @Before
+    public void setup() {
+        Framework.getProperties().put("org.nuxeo.vision.google.credential", "/Users/thibaud/Nuxeo-Misc/google-vision-nuxeo-credentials.json");
```

Comment:
Path to the credential file is hardcoded. The path must be set in the NUXEO_GOOGLE_APPLICATION_KEY environment variable (see README)